whitepaper/whitepaper_zh-CN.md

```diff
@@ -0,0 +1,980 @@
+Nonetheless, arguably a more interesting approach would be to consider cross-smart contract execution functionality or another automatic way of migration.
+
+References: <br>
+E Scheid and Burkhard Rodrigues, B Stiller. 2019. Toward a policy-based blockchain agnostic framework. 16th IFIP/IEEE International Symposium on Integrated Network Management (IM 2019) (2019)
+Philipp Frauenthaler, Michael Borkowski, and Stefan Schulte. 2019. A Framework for Blockchain Interoperability and Runtime Selection.
+
+### 2.8.1 Blockchain Data Migration 区块链数据整合
+Data migration corresponds to capture the set or subset of data assets (information, in the form of bytes) on a source blockchain, and construct a representation of those in a target blockchain. Note that the models underlying both blockchains do not need to be the same (e.g., world state model in Hyperledger Fabric vs account model in Ethereum).
+To migrate data, it should be possible to capture the necessary information from the source blockchain and to write it on the target blockchain. The history of information should also be migrated (i.e., the updates over the elements considered information).
+
+### 2.8.2 Blockchain Smart Contract Migration 区块链智能合约整合
+The task of migrating a smart contract comprises the task of migrating data. In specific, the information should be accessible and writeable on another blockchain. Additionally, the target blockchain's virtual machine should support the computational complexity of the source blockchain (e.g., one cannot migrate all Ethereum smart contracts to Bitcoin, but the other way around is feasible).
+
+Automatic smart contract migration yields risks for enterprise blockchain systems, and thus the solution is non-trivial.
+
+### 2.8.3 Semi-Automatic Blockchain Migration 半自动区块链整合
+
+By expressing my preferences in terms of functional and non-functional requirements, Hyperledger Cactus can recommend a set of suitable blockchains, as the target of the migration.
+Firstly, I could know in real-time the characteristics of the target blockchain that would influence my decision.
+For instance, the platform can analyze see the cost of writing information to Ethereum, the exchange rate US dollar - Ether, the average time to mine a block, the transaction throughput, and the network hash rate ((Frauenthaler et al., 2019).
+Based on that, the framework proposes a migration, with indicators such as predicted cost, predicted time to complete migration and the likelihood of success.
+As Ethereum does not show a desirable throughput, I choose Polkadot's platform. As it yields higher throughput, I then safely migrate my solution from Fabric to Polkadot, without compromising the solution in production. 
+ This feature is more useful regarding public blockchains.
+
+
+<div style="page-break-after: always; visibility: hidden"><!-- \pagebreak --></div>
+
+
+# 3. 软件设计
+
+## 3.1. 原则
+
+### 3.1.1. 广泛的支持
```

Comment:
这一段开始怎么不附带原文了？

Reply:
翻译内容还打算和Fabric一样，只保留中文，前边的内容因为还没翻译完，所以英文没有删。

Reply:
This commit only contains the translation  of part3, other parts are translating now. The paper is too long, I am translating it by part.

Reply:
我觉得原文不长的话第一个commit可以先留着。

Reply:
> This commit only contains the translation of part3, other parts are translating now. The paper is too long, I am translating it by part.

@stone-ch Thank you very much for the translation! Do you plan on doing all of it in one PR or submit multiple PRs whenever you get some time to finish each section? I'm asking because we have two ways to go right now:
1. We LGTM this now and then merge it in and then you need to open another PR with other sections translations later.
2. We convert this to a draft PR now and make it a ready for review status (e.g. non-draft) PR whenever you are done making all your changes.

I'm completely fine with any one of those options, we just need to coordinate and pick one or the other to keep things tidy so let me know which one would you prefer please and thanks again for the translation, this is great!

Reply:
@petermetz  Please convert this to a draft PR now, the first version of the translation will be finished this week, then you can merge the full translations. After that I will update the translation when the whitepaper updated.

Reply:
> @petermetz Please convert this to a draft PR now, the first version of the translation will be finished this week, then you can merge the full translations. After that I will update the translation when the whitepaper updated.

@stone-ch Got it, thank you. Converting to draft for now.